### PR TITLE
Added Screens and all Use Cases for Invitation + Collaborative Task

### PR DESCRIPTION
**Pull request by sshmuylovich**

### CODE REVIEW

**Review by celinelumqr — APPROVED** [review]
mostly adheres to clean architecture and SOLID and well documented

File: src/main/java/use_cases/collaborative_task_management/collaborative_task_edit_use_case/CollaborativeTaskEditRequestModel.java [hunk 1]
```diff
@@ -1,46 +1,44 @@
 package use_cases.collaborative_task_management.collaborative_task_edit_use_case;
 
-import entities.*;
+import entities.StudentUser;
```
Comment (CC-3636) [inline src/main/java/use_cases/collaborative_task_management/collaborative_task_edit_use_case/CollaborativeTaskEditRequestModel.java:3]: small issue: importing entities into Request model violates Common Closure & Single Responsibility Principle--see page 198 on the pdf of the CA book.

**Review by CC-3636 — CHANGES_REQUESTED** [review]
Overall great documentation but some changes needed in CollaborativeTaskInfoRetriever and CollaborativeTaskEditRequestModel in order to adhere to Clean Architecture. Otherwise looks good.

File: src/main/java/use_cases/collaborative_task_management/collaborative_task_edit_use_case/CollaborativeTaskEditRequestModel.java (hunk 1 above)
Comment (sshmuylovich) [reply src/main/java/use_cases/collaborative_task_management/collaborative_task_edit_use_case/CollaborativeTaskEditRequestModel.java:3]: Changed!

File: src/main/java/screens/collaborative_task_management/collaborative_task_edit_delete_screens/CollaborativeTaskInfoRetreiver.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package screens.collaborative_task_management.collaborative_task_edit_delete_screens;
+
+import entities.CollaborativeTask;
+import entities.StudentUser;
```
[the remark this answers is not in the document]
Comment (sshmuylovich) [reply src/main/java/screens/collaborative_task_management/collaborative_task_edit_delete_screens/CollaborativeTaskInfoRetreiver.java:4]: There are other screens in the program that are like this  that have already been merged.

**Review by celinelumqr — DISMISSED** [review]

**Review by celinelumqr — APPROVED** [review]

**Review by yyutiw — APPROVED** [review]

**Review by alyson647 — APPROVED** [review]
Looks good!

**Pull request merged**